tests/Stripe/Service/AbstractServiceTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Stripe\Service;
+
+class AbstractServiceTest extends \Stripe\TestCase
+{
+    const TEST_RESOURCE_ID = "25OFF";
```

Comment:
really minor but can we do `RESOURCE_ID` or similar instead. Trying to get rid of the fact that you can do your own ids on some objects

Reply:
👍